src/main/resources/defaults.yaml

```diff
@@ -0,0 +1,380 @@
+a:link:
+  - color: (internal value)
```

Comment:
(internal value) is not really a default. The browser may decide. So I would remove it here.

Reply:
Good catch, missed that :-D